packages/react/storybook/.storybook/main.ts

```diff
@@ -14,6 +14,22 @@ const config: StorybookConfig = {
     name: '@storybook/react-vite',
     options: {},
   },
+  async viteFinal(config, { configType }) {
```

Comment:
Do we need this now?

@BogdanDanilescu

Reply:
I would say let's keep this because this is what enables hot reloading, so it saves a lot of time for us